pkg/broker/broker.go

```diff
@@ -101,16 +101,15 @@ func (a AnsibleBroker) getServiceInstance(instanceUUID uuid.UUID) (*apb.ServiceI
 
 }
 func (a AnsibleBroker) Login() error {
-	a.log.Debug("Retrieving serviceaccount token")
-	token, err := ioutil.ReadFile("/var/run/secrets/kubernetes.io/serviceaccount/token")
+	clientConfig, err := k8srestclient.InClusterConfig()
```

Comment:
This looks like a larger cleanup patch is needed.  To get the client you would call ```apb.Client.ClusterClient```.  The problem is the client is created during a provision, bind, unbind, or deprovision.  So we either stay with what you have and make note of the tech debt (I did already in the post TP etherpad) or create the clients before the broker.

https://github.com/openshift/ansible-service-broker/blob/master/pkg/apb/provision.go#L61